commands/execute.py

```
@@ -147,97 +146,599 @@ def get_image_filename(i):
+    # 最適化モジュールがない場合のフォールバック
+    print("注意: 最適化モジュールが利用できません。標準モードで実行します。")
+    print("内部でシンプルなフォールバック実装を使用します。")
+    OPTIMIZE_AVAILABLE = True
+    
+    # シンプルなフォールバック実装
+    def is_apple_silicon():
+        try:
+            if platform.system() != "Darwin":
+                return False
+            return platform.processor() == "arm" or "Apple" in subprocess.check_output(["sysctl", "-n", "machdep.cpu.brand_string"]).decode("utf-8").strip()
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)